src/Elastic.Apm/Model/Span.cs

```diff
@@ -232,6 +232,10 @@ public override string ToString() => new ToStringBuilder(nameof(Span))
 			{ nameof(IsSampled), IsSampled }
 		}.ToString();
 
+		public Label GetLabel(string key) => Context.InternalLabels.Value.InnerDictionary.ContainsKey(key)
+			? Context.InternalLabels.Value.InnerDictionary[key]
+			: null;
```

Comment:
```suggestion
		public Label GetLabel(string key)
		{
		    Context.InternalLabels.Value.InnerDictionary.TryGetValue(key, out var label);
		    return label;
		}
```

Reply:
With `TryGetValue` , the lookup is performed once

Reply:
Agreed - I'm using `TryGetValue` with some other changes described [here](https://github.com/elastic/apm-agent-dotnet/pull/1057#issuecomment-731802674).